standard_readme.md

```diff
@@ -0,0 +1,75 @@
+# Name of the project
+
+Description of the project. What is it solving? Who are the users?
+
+URL to the project
+Asana:
+URL to Abtion's own related git repositories (frontend / backend / admin area / microservices)
+Harvest:
+Travis:
+Client name, and if possible, contact details including IT contact details
+Name, and a short description of any services that the project is using (error tracking, activity monitoring, log registry, email service, etc). Include where to get the credentials. E.g., 1Password under admin+project@abtion.com.
+
+This section should include any business related explanation that helps understand the context of the project. It could be an excellent idea to include a dictionary of terms, a color legend or a user roles explanation.
+
+# Development
+
+## Requirements
+
+System requirements, library dependencies, etc.
+
+## Set up the app the first time
+
+Instructions on how to set up the app the first time.
+
+## Run the app
+
+Instructions on how to run the app.
+
+## Run the tests
+
+## Useful commands
+
+## Download production / staging DB
+
+Instructions on how to download the production/staging database.
+
+## Workflow
+
+Workflow agreed by the team. Include branch strategy, PR strategy, branch ownership, etc.
+
+## Documentation
+
+Where to find the documentation.
+
+# Production
+
+## Automatic deploy
+
+How does automatic deploy happen?
```

Comment:
'How do automatic deploys happen?' sounds a little... I dunno softer? it rolls better of the tongue I'd say 🤔